Script/server.js

```diff
@@ -69,7 +69,8 @@ function loadReporterFromServer(ticketId)
     
         return response;
 }
-function getActiveUsers()
+
+function GetActiveUsersFromServer()
```

Comment:
Fixing, methods should not begin with a capital letter

Reply:
fixed